fppanalysis/running_moments.py

```diff
@@ -137,3 +136,23 @@ def run_norm(S, radius, T=None, return_run_moment=False):
         res = res[0]
 
     return res
+
+
+def run_norm_window(cut_off_freq, time):
+    """Computes number of data points of window size used in running
+    normalization given a cut off frequency and the data time array. This
+    window size will be multiplied with 2 in running normalization and running
+    moments functions.
+
+    Returns
+        window: length of window given in number of data points
+    """
+    import numpy as np
+
+    dt = np.diff(time)[0]
+    t_run_mean = 1 / cut_off_freq
+    samples = ((t_run_mean / dt) - 1) / 2
+    # Window size is mulitplied with 2 in run_norm function, and therefore
+    # not multiplied with 2 here
```

Comment:
I am confused by this comment, I don't see that run_norm uses this function?
Also, if this function is only meant to be used internally (is that the case?), then it should be private (add _ prefix)